### ISSUE TITLE
bug(mat-table): ChromeVox screen-reader highlights multiple table cells at a time

### ISSUE THREAD

**Issue by pieman72** [angular/components#28347]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

When navigating with screen reader controls only one table cell should be highlighted at a time, however multiple cells are highlighted.

### Reproduction

StackBlitz link: https://stackblitz-starters-fcezcx.stackblitz.io

Steps to reproduce:
1. Once the StackBlitz has loaded, click anywhere within the rendered demo.
2. Activate ChromeVox (CTRL + ALT + z)
3. Have ChromeVox select the table (Shift + Search + t), and you'll see that the first two table headers are highlighted together and the third is highlighted separately.
4. Now navigate around the table (CTRL + ALT + Search + [arrows]), and you'll see that multiple table cells are always highlighted

### Expected Behavior

When navigating around the table, only one table cell at a time should be highlighted by ChromeVox.

### Actual Behavior

Generally, the focused cell is highlighted together with the cell to its immediate right (if there is one). Also, all cells in the same row as the focused cell are highlighted separately.

**Note:** focusing and voicing are correct; it's only the highlighting that's wrong.

### Environment

- Angular: **17.0.8**
- CDK/Material: **17.0.4**
- Browser(s): **Chrome**
- Operating System: **ChromeOS**

**Comment by zarend** [comment]
Hello, I can also reproduce this issue on [ARIA Authoring Practices Guide](https://www.w3.org/WAI/ARIA/apg/patterns/table/examples/table/). I believe this is an issue with ChromeOS, and I don't think this is an issue with Angular components.

 - Chrome/ChromeOS Version 123.0.6312.112 (Official Build) (64-bit)

@essjay05 would you like to file this issue report on ChromeOS?

Best Regards,

Zach

**Comment by essjay05** [comment]
Sounds good! Will do.
  
  
  
  
  
  
  

  
  
>
